yarn.lock

```diff
@@ -5249,6 +5261,15 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@calcom/skype@workspace:packages/app-store/skype":
```

Comment:
I have to check in the `yarn.lock` so that we include `@urql/core`, but I don't know why the lock file also contains changes unrelated to urql.

A review is necessary from someone familiar with the yarn set up (cc: @emrysal).
FWIW I did try all of the scripts & recommendations provided in the handbook by @zomars. But I always end up with these changes.